### PR TITLE
feat(statics): add honey & USDC token for bera

### PR DESCRIPTION
**Pull request by adarsh312**
Ticket: WIN-3597

We are adding the ERC20 tokens (Honey and USDC) for BERA

### CODE REVIEW

File: modules/statics/src/coins.ts [hunk 1]
```diff
@@ -18374,6 +18392,22 @@ export const coins = CoinMap.fromCoins([
...
+    'tbera:honey',
+    'Bera Testnet Honey',
+    18,
+    '0x0E4aaF1351de4c0264C5c7056Ef3777b41BD8e03',
+    UnderlyingAsset['tbera:honey']
+  ),
+  tberaErc20(
+    '7319878e-96ca-4a97-bad8-b4b9d040b94b',
+    'tbera:usdc',
+    'Bera Testnet usdc',
```
Comment (ajays97) [inline modules/statics/src/coins.ts:18406]:
```suggestion
    'Bera Testnet USDC',
```

File: modules/statics/src/coins.ts [hunk 2]
```diff
@@ -18366,6 +18366,24 @@ export const coins = CoinMap.fromCoins([
     '0xbda130737bdd9618301681329bf2e46a016ff9aa',
     UnderlyingAsset['bera:bgt']
   ),
+  beraErc20(
+    'ca86baf8-fcc6-40ff-9d65-08db513a131e',
+    'bera:honey',
+    'Honey Token',
+    18,
+    // TODO: the mainnet contract address is still not available, adding placeholder here
```
Comment (ajays97) [inline modules/statics/src/coins.ts:18374]:
```suggestion
    // TODO: the mainnet contract address is still not available, adding placeholder here, WIN-3597
```

File: modules/statics/src/coins.ts [hunk 3]
```diff
@@ -18366,6 +18366,24 @@ export const coins = CoinMap.fromCoins([
...
+    '0xbda130737bdd9618301681329bf2e46a016ff9ab',
+    UnderlyingAsset['bera:honey']
+  ),
+  beraErc20(
+    '31cdb51f-3bcc-489e-8b35-8e074169c573',
+    'bera:usdc',
+    'USDC Token',
+    18,
+    // TODO: the mainnet contract address is still not available, adding placeholder here
```
Comment (ajays97) [inline modules/statics/src/coins.ts:18383]:
```suggestion
    // TODO: the mainnet contract address is still not available, adding placeholder here, WIN-3597
```

**Review by ajays97 — DISMISSED** [review]
nits, but LGTM!

**Comment by adarsh312** [comment]
@ajays97 Fixed the issue, can you please help with a review

**Review by ajays97 — APPROVED** [review]

**Pull request merged**